microcloud/test/includes/microcloud.sh

```diff
@@ -350,16 +404,17 @@ reset_snaps() {
     # These are set to always pass in case the snaps are already disabled.
     echo "Disabling LXD and MicroCloud for ${name}"
     lxc exec "${name}" -- sh -c "
-      if pidof -q lxd ; then
-        kill -9 \$(pidof lxd)
-      fi
+      rm -rf /var/snap/lxd/common/lxd.debug
+      rm -rf /var/snap/microcloud/common/microcloudd.debug
+      rm -rf /var/snap/microcloud/common/microcloud.debug
 
-      snap disable lxd > /dev/null 2>&1 || true
-
-      if pidof -q microcloud ; then
-        kill -9 \$(pidof microcloud)
-      fi
+      for app in lxd lxd.debug microcloud microcloud.debug ; do
```

Comment:
Sorry if I am missing something but shouldn't this line include `microcloudd microcloudd.debug`? I guess the daemon should be killed here.

Reply:
Yes, good catch. `microcloud` is just the client.